### DISCUSSION TITLE
how comes minio is suddenly using http instead of https for port 8443?

### DISCUSSION THREAD

**Discussion by harridu** [minio/minio#18719] [Q&A] [answered]
how comes minio is suddenly using http instead of https for port 8443? Is this just me? 

I am using letsencrypt certs as before. Since the upgrade to the minio version released on Dec 23rd it seems to be broken.

**Comment by klauspost** [comment]
https://github.com/minio/minio/issues/18720

**Comment by harshavardhana** [comment, marked as answer]
You need to use --certs-dir/-S instead of -C, -C command has been long deprecated it was finally removed.
